packages/mjml-button/src/index.js

```diff
@@ -56,6 +56,8 @@ export default class MjButton extends BodyComponent {
       table: {
         'border-collapse': 'separate',
         width: this.getAttribute('width'),
+        // Ensure that wrong line-height wouldn't be inherited
```

Comment:
Can we remove the comment here ?

Reply:
Probably, i'm personally prefer comments to git-blaming to trace the reason why code were added

I can remove it, if you want